blocks/serialization.py

```diff
@@ -215,10 +218,13 @@ def secure_dump(object_, path, dump_function=dump, **kwargs):
 
     """
     try:
+        logger.debug("Dump object to a temporaty file")
```

Comment:
Oh, sorry, here it actually makes more sense to use "dumping". I meant that "dump is finished" is probably proper English.